dashboard/api.py

```diff
@@ -498,8 +502,10 @@ def get_student_current_grades(user, edx_client):
     refresh_delta = now - datetime.timedelta(hours=REFRESH_GRADES_CACHE_HOURS)
 
     with transaction.atomic():
+        enrolled_curses = models.CachedEnrollment.objects.filter(user=user).exclude(data=None).values_list(
+            "course_run__edx_course_key", flat=True)
```

Comment:
we create an entry in the cache for each user and each course run no matter if there is data or not.
This because it is easier to understand when the cache needs to be refreshed or not.
But this also means that the majority of the entries have `data=null`